Faust Siu Proj 2

```diff
@@ -169,13 +183,15 @@ proc sgplot data=schoenfeld_los;
     loess x = loglos y = schsho / clm;
```

Comment:
I don't know what happened here, but there needs to be an SG PLOT for age and sho with each outcome variable. 

Code: 
/*SGPLOT SCHOENFELD AGE los*/
proc sgplot data = schoenfeld_los; 
	loess x = loglos y = schage/ clm;
	title 'Age Residuals by Time';
run;

/*SGPLOT SCHOENFELD SHO los*/
proc sgplot data = schoenfeld_los; 
	loess x = loglos y = schsho / clm;
	title 'SHO Residuals by Time';
run;

/*SGPLOT SCHOENFELD AGE lenfol*/
proc sgplot data=schoenfeld_lenfol; 
	loess x = loglenfol y = schage / clm;
	title 'Age Residuals by Time';
run;

/*SGPLOT SCHOENFELD SHO lenfol*/
proc sgplot data=schoenfeld_lenfol; 
	loess x = loglenfol y = schsho / clm;
		title 'SHO Residuals by Time';
run;